ramls/inventory-hierarchy/inventory-instance-records.json

```diff
@@ -0,0 +1,557 @@
+                      "description": "The name of the institution, the first-level location unit, this (shelf) location belongs to",
+                      "type": "string"
+                    }
+                  },
+                  "required": [
+                    "name",
+                    "campusName",
+                    "libraryName",
+                    "institutionName"
+                  ]
+                }
+              },
+              "required": [
+                "permanentLocation",
+                "temporaryLocation"
+              ]
+            },
+            "callNumber": {
+              "description": "Call Number is an identifier assigned to a holding",
+              "type": "object",
+              "properties": {
+                "prefix": {
+                  "description": "Prefix of the call number on the holding level",
+                  "type": "string"
+                },
+                "suffix": {
+                  "description": "Suffix of the call number on the holding level",
+                  "type": "string"
+                },
+                "typeId": {
+                  "description": "Unique ID for the type of call number on a holdings record",
+                  "type": "string",
+                  "$ref": "../uuid.json"
+                },
+                "typeName": {
+                  "description": "Name of the call number type",
+                  "type": "string"
+                },
+                "callNumber": {
+                  "description": "Call Number identifier assigned to a holding",
+                  "type": "string"
+                }
+              },
+              "required": [
+                "prefix",
+                "suffix",
+                "typeId",
+                "typeName",
+                "callNumber"
+              ]
+            },
+            "shelvingTitle": {
+              "description": "Indicates the shelving form of title",
+              "type": "string"
+            },
+            "acquisitionFormat": {
+              "description": "Format of holdings record acquisition",
+              "type": "string"
+            },
+            "acquisitionMethod": {
+              "description": "Method of holdings record acquisition",
+              "type": "string"
+            },
+            "receiptStatus": {
+              "description": "Receipt status (e.g. pending, awaiting receipt, partially received, fully received, receipt not required, and cancelled)",
+              "type": "string"
+            },
+            "electronicAccess": {
+              "description": "Aggregated electronic access from holdings",
+              "type": "array",
+              "items": [
+                {
+                  "type": "object",
+                  "properties": {
+                    "uri": {
+                      "description": "Electronic access URI",
+                      "type": "string",
+                      "$ref": "../url.json"
```

Comment:
The underlying holdingsrecord.json uri doesn't have such a validation pattern: https://github.com/folio-org/mod-inventory-storage/blob/v19.3.1/ramls/holdingsrecord.json#L57-L60
How can you ensure that an uri that fails the validation pattern is not returned in inventory-hierarchy/inventory-instance-records.json?

I suggest to completely remove `"$ref": "../url.json"`. If there is a need for validation a separate Jira and a separate pull request should be opened to discuss the requirements.

Reply:
> I suggest to completely remove "$ref": "../url.json"

I will remote it, thank you for noticing